src/adapters/serverbid.js

```diff
@@ -44,43 +48,67 @@ const ServerBidAdapter = function ServerBidAdapter() {
 
   baseAdapter.callBids = function(params) {
     if (params && params.bids && utils.isArray(params.bids) && params.bids.length) {
-      const data = {
-        placements: [],
-        time: Date.now(),
-        user: {},
-        url: utils.getTopWindowUrl(),
-        referrer: document.referrer,
-        enableBotFiltering: true,
-        includePricingData: true
+      window[SMARTSYNC_CALLBACK] = function() {
+        _callBids(params);
```

Comment:
for performance reasons we don't want to delay the auctions, but rather initiate the auctions, then do the syncs.  I realize this means the sync won't be there for the first auction(s), but it will for subsequent auctions on the site.